src/pages/Login.vue

```diff
@@ -162,12 +162,12 @@
         :buttonClass="resendOTPButtonClass"
         class="mt-2"
         :isDisabled="submitOTPIconConfig.enabled"
-        v-if="requestedOtp && !resentOtp"
+        v-if="requestedOtp && !counting"
         data-test="resendOTP"
       ></icon-button>
-      <!-- text to show when OTP has been resent -->
-      <p v-if="resentOtp" class="text-center mt-2">
-        {{ $t("login.otp.resent") }}
+      <!-- text to show when OTP will be resent -->
+      <p v-if="counting" class="text-center mt-2">
+          {{$t("login.otp.timer")}} {{ counter }} {{$t("login.otp.secs")}}
       </p>

```

Comment:
you do not need to create another element for showing the resent text - you can simply update the text of the resend button + add the classes `opacity-50` and `cursor-pointer-not-allowed` to the resend button class if the timer is being shown!

Also, we decided on using brackets right? - `Resend OTP (60 secs)` instead of `Resend OTP in 60 seconds`

Reply:
It is looking better without brackets so that's why I didn't do that.